### PR TITLE
Common config update

### PR DESCRIPTION
**Pull request by geans-pin**
##### Work item tracking
- Microsoft ADO **(number only)**:

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: device/broadcom/x86_64-broadcom_common/x86_64-broadcom_b27/broadcom-sonic-td3.config.bcm [hunk 1]
```diff
@@ -0,0 +1,14 @@
+mem_cache_enable=1
```
Comment (ZhaohuiS) [inline device/broadcom/x86_64-broadcom_common/x86_64-broadcom_b27/broadcom-sonic-td3.config.bcm:1]: @geans-pin b27 is TD3, b37 is TD3, are they same?

Comment (geans-pin) [reply device/broadcom/x86_64-broadcom_common/x86_64-broadcom_b27/broadcom-sonic-td3.config.bcm:1]: No, one is TD3 X2, one is TD3 X3

**Comment by geans-pin** [comment]
[Uploading common config UT.pdf…]()

**Comment by geans-pin** [comment]
> /azpw ms_conflict

**Comment by geans-pin** [comment]
close this one and create a new one 20802

**Pull request closed without merging**